TSPL.docc/ReferenceManual/Attributes.md

```diff
@@ -1304,6 +1304,53 @@ can increase your binary size and adversely affect performance.
   because of the larger symbol table slowing dyld down.
 -->
 
+### preconcurrency
+
+Apply this attribute to a declaration,
+to suppress strict concurrency checking.
+
+On an import declaration,
+this attribute reduces the strictness of concurrency checking
+for code that uses types from the imported module.
+Specifically,
+types from the imported module
+that aren't explicitly marked as either sendable or nonsendable
```

Comment:
```suggestion
that aren't explicitly marked as nonsendable
```

Sendable types can always be used in a context that requires a sendable type, so I think you can simplify the above a little.